fenix/app/src/main/java/org/mozilla/fenix/tabstray/browser/compose/ReorderableList.kt

```diff
@@ -266,17 +276,27 @@ private fun LazyListState.findItem(offset: Float) =
  * @param reorderState List reordering state used for dragging callbacks.
  * @param shouldLongPressToDrag Whether or not an item should be long pressed to start the dragging gesture.
  */
-fun Modifier.detectVerticalPressAndDrag(
+fun Modifier.detectListPressAndDrag(
     listState: LazyListState,
     reorderState: ListReorderState,
     shouldLongPressToDrag: Boolean,
-): Modifier = pointerInput(listState, shouldLongPressToDrag) {
+): Modifier = this then pointerInput(listState, shouldLongPressToDrag) {
```

Comment:
the inline `then` is pretty neat. TIL

Reply:
@MozillaNoah ➕ I initially thought it was a typo 🙃